config-base.js

```diff
@@ -108,19 +116,29 @@ module.exports = {
     'unicorn/catch-error-name': 'off',
     'unicorn/consistent-function-scoping': 'off',
     'unicorn/no-abusive-eslint-disable': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-await-expression-member': 'off',
     'unicorn/no-fn-reference-in-iterator': 'off',
     'unicorn/no-nested-ternary': 'warn',
     'unicorn/no-null': 'off',
     'unicorn/no-process-exit': 'off',
     'unicorn/no-reduce': 'warn',
+    'unicorn/numeric-separators-style': 'off',
```

Comment:
I don't know why we'd turn this off. It seems like a useful rule to me.

Reply:
week-later follow-up, it was disabled because it will warn on values like `40000` so it'd be enforcing `40_000` which I know many services and tests don't strictly follow this

Reply:
That seems good though and I'm pretty sure ESLint could auto-fix that

Reply:
just re-bumping the convo after testing in a couple more services, but we often separate numbers by cents as well which gets flagged up by this rule, I'm definitely leaning more towards having this rule off because of this.

For example, `20_000_00` is how we denote $20,000 in cents but it's technically incorrect usage of numeric separators

Reply:
Yeah, that makes sense